benchmarks/torchbench/benchtest.yaml

```diff
@@ -1,8 +1,8 @@
 defaults: &defaults
   dirs:
     code: code/{name}
-    venv: null
-    data: data
+    venv: venv/{name}
+    data: data/{name}
```

Comment:
Technically, they are not installed in parallel. I think you can just use `venv/torchbench`. I might add a `group` key to configuration so that benchmarks can be more easily grouped together for dependencies.

It shouldn't be `data/{name}` generally because datasets can be shared. Ultimately it doesn't really matter though because the torchbench data is obtained with git-lfs.

Reply:
Hehe I know but I'm `echo | xargs | milabench install --select` them to speed the install. In the install process I download and extract the git-lfs bins and if multiple process do the same thing on the same files, I fear undefined behaviour when executing the bins. I could write a file lock in the `.sh` script but is there a way for milabench to know that it only need to execute the git-lfs once for all the models?

Reply:
maybe I should download / extract in `code` instead

Reply:
Each bench should have its own `code` directory currently because it sets a sentinel in there to know whether it's done with the install or not. In practice that means only the first one would install.

Reply:
I'm not sure if I understand correctly but with the current config:
```
defaults: &defaults
  dirs:
    code: code/{name}
    venv: venv/{name}
    data: data
    runs: runs
```
does create multiple `code/MODEL` dirs and git-lfs is downloaded and extracted in each of them. Let me know if that is a desired behaviour

Reply:
Eh, it'll work. We can try to optimize that after the RFP, maybe.